scripts/process_dgs_videos.py

```
@@ -1,219 +1,297 @@
-
-    # Save to file
-    os.makedirs(os.path.dirname(output_path), exist_ok=True)
-    with open(output_path, 'w') as f:
-        json.dump(training_data, f, indent=2)
-
-    print(f"Saved {len(training_data['samples'])} samples to {output_path}")
+def save_output(samples: List[Dict[str, Any]], output_path: str, split_output: bool, videos_dir: str):
+    # Bulk save
+    if output_path:
+        data = {"samples": [{"label": s["label"], "landmarks": s["landmarks"]} for s in samples]}
+        with open(output_path, 'w') as f: json.dump(data, f)
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)